src/components/avatar/avatar.component.ts

```diff
@@ -54,6 +57,11 @@ export default class SlAvatar extends ShoelaceElement {
     this.hasError = false;
   }
 
+  private onImageLoadError() {
```

Comment:
nit: we try to use this convention for handlers

```suggestion
  private handleImageLoadError() {
```

Reply:
👍 Ty for pointing that out. Updating.